crates/_support/objkv/tests/bucket.py

```
@@ -0,0 +1,111 @@
+        method=method,
+        headers={
+            "Authorization": (
+                f"AWS4-HMAC-SHA256 Credential={KEY}/{scope}, "
+                "SignedHeaders=host;x-amz-content-sha256;x-amz-date, "
+                f"Signature={sig}"
+            ),
+            "x-amz-content-sha256": empty,
+            "x-amz-date": amz,
+        },
+    )
+    return urllib.request.urlopen(req, timeout=30).read()
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)